e2e/pages/adf/content-services/documentListPage.ts

```diff
@@ -0,0 +1,117 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { browser, by, element } from 'protractor';
+import { DataTablePage } from '../dataTablePage';
+import { Util } from '../../../util/util';
+
+export class DocumentListPage {
+
+    dataTable = new DataTablePage();
+    deleteContentElement = element(by.css('button[data-automation-id*="DELETE"]'));
+    metadataAction = element(by.css('button[data-automation-id*="METADATA"]'));
+    versionManagerAction = element(by.css('button[data-automation-id*="VERSIONS"]'));
+    moveContentElement = element(by.css('button[data-automation-id*="MOVE"]'));
+    copyContentElement = element(by.css('button[data-automation-id*="COPY"]'));
+    lockContentElement = element(by.css('button[data-automation-id="DOCUMENT_LIST.ACTIONS.LOCK"]'));
+    downloadContent = element(by.css('button[data-automation-id*="DOWNLOAD"]'));
+    actionMenu = element(by.css('div[role="menu"]'));
+    optionButton = by.css('button[data-automation-id*="action_menu_"]');
+    rows = by.css('div[id="document-list-container"] div[class*="adf-datatable-body"] div[class*="adf-datatable-row"]');
+
+    dataTablePage() {
+        return new DataTablePage();
+    }
+
+    deleteContent(content) {
```

Comment:
This kind of methods, related to an action like: deleteContent, metadataContent, etc, needs to be at contentServicesPage level(demo-shell level). Only a general method like: clickOption(option) should be on component level, but shouldn't be on documentList, but on datatable.